### PR TITLE
first simple routes

### PR DESCRIPTION
**Pull request by dmbavykin**

### CODE REVIEW

File: lib/racker.rb [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+  def welcome_handler(data)
+    Rack::Response.new do |response|
+      response.set_cookie('user_id', data[:user_id].to_json)
+      path = if data[:game]
+               refresh_game_data(data[:game], response, data[:user_id])
+               '/game'
+             else
+               '/difficulty'
+      end
```
Comment (kirillshevch) [inline lib/racker.rb:51]: rubocop?

File: Gemfile [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+ruby '2.4.1'
 gem 'bundler', '~> 1.16'
-gem 'pry', '~> 0.11.3'
-gem 'rspec', '~> 3.0'
-gem 'rubocop', '~> 0.51.0'
+gem 'codebreaker_Bavykin', '~> 0.1.4'
+gem 'materialize-sass', '~> 0.100.2'
 gem 'rack', '~> 2.0', '>= 2.0.3'
+gem 'rack-test', '~> 0.8.2'
+gem 'rspec', '~> 3.7'
```
Comment (kirillshevch) [inline Gemfile:12]: gems related to tests should be in `group :test`

File: lib/models/welcome.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  def to_cypher_user_name
+    @cyphered_name ||= Digest::MD5.hexdigest("#{@name}#{@ip}").to_sym
+  end
+
+  def old_user?
+    @users = dataOrHash('users')
+    @users.key?(to_cypher_user_name)
+  end
+
+  def dataOrHash(name)
```
Comment (kirillshevch) [inline lib/models/welcome.rb:35]: all methods in ruby should be named in `snake_case`

File: lib/racker.rb (hunk 1 above)
Comment (dmbavykin) [reply lib/racker.rb:51]: Yes. I used "rubocop -a"

**Pull request merged**